tests/capture/test_nested_plxpr.py

```diff
@@ -186,6 +186,54 @@ def workflow(x):
         out = jax.core.eval_jaxpr(plxpr.jaxpr, plxpr.consts, 0.5)
         assert qml.math.isclose(out, qml.math.sin(-(0.5 + 0.3)))
 
+    def test_dynamic_shape_input(self):
+        """Test that the adjoint transform can accept arrays with dynamic shapes."""
+        jax.config.update("jax_dynamic_shapes", True)
+        try:
```

Comment:
Probably best to put the `enable_disable_dynamic_shapes` fixture in `conftest.py` and using that here. I anticipate more tests for dynamic shapes in the future.

Reply:
Using a fixture now.